app/views/newflow/password_management/forgot_password_form.html.erb

```diff
@@ -0,0 +1,73 @@
+<div id="login-signup-form" class="forgot-password-form">
+    <div class="content back-arrow-container">
+        <%= render partial: 'newflow/tab_group', locals: { active_class: 'login--active'} %>
+
+        <%= newflow_login_signup_card(
+                    classes: "login-page",
+                    header: I18n.t(:"login_signup_form.reset_my_password")) do %>
+            <% lev_form_for :forgot_password_form, url: send_reset_password_email_path do |f| %>
+                <%
+                    form_helper = NewflowFormHelper::Newflow.new(
+                        f: f,
```

Comment:
would really love to see variables fully spelled out and not "f".   yeah..we know what "f" means, but in general, always good to never use "f", or "x", or "z" sorts of things.

Reply:
I couldn't agree more! I did change some of them already, where `f` and `fh` are used instead of `form` and `form_helper` but I missed this one and I admit that there are still a few more left to spell out, it's just too many to fix at once. I could do it though, if you insist.